src/App.js

```diff
@@ -1,17 +1,97 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
 import './App.css';
+import Pokemon from './components/pokemon/Pokemon';
+import Button from './components/button/Button';
 import axios from 'axios';
+import pikachu from './assets/pikachu.gif';
+import logo from './assets/logo.jpg';
 
 function App() {
-  return (
-      <>
-    <div>
-      Begin hier met de opdracht!
-    </div>
-          <button type="button">Vorige</button>
-          <button type="button">Volgende</button>
-      </>
-  );
+    const [pokemonArray, setPokemonArray] = useState([]);
+    const [endpoint, setEndpoint] = useState('https://pokeapi.co/api/v2/pokemon/');
+    const [isLoading, setLoading] = useState(false);
+    const [catchError, setCatchError] = useState(null);
+
+
+    useEffect(() => {
+        async function fetchEmAll() {
+            setLoading(true);
+            setCatchError(false);
+
+
+            try {
+                const response = await axios.get(endpoint);
+                console.log(response);
+                setPokemonArray(response.data);
+
+            } catch (e) {
+                console.error(e);
+                setCatchError(true);
+
+            }
+            setLoading(false);
+
+        }
+
+        fetchEmAll();
+    }, [endpoint]);
+
+    return (
+        <>
+            <section className="logo">
+                <img src={logo} alt="pokemon logo" width="400px"/>
+
+            </section>
+            <div className="poke-deck">
+
+                <>
+                    {Object.keys(pokemonArray).length > 0 &&
```

Comment:
Als `pokemonArray` altijd een array is, dan kun je dit wat simpeler doen
```suggestion
                    {pokemonArray.length > 0 &&
```